### ISSUE TITLE
[bug] cleanyerreads workflow fails in bactopia 3.0.1 

### ISSUE THREAD

**Issue by pvanheus** [bactopia/bactopia#501]
**Description**
`cleanyerreads` workflow fails in bactopia 3.0.1. This works (with exactly the same data and bactopia flags) in version 3.0.0.

**Steps to Reproduce**
```
bactopia --wf cleanyerreads --samples sample-sheet.tsv --outdir cleaned_reads

2024-04-03 13:13:51 INFO     2024-04-03 13:13:51:root:INFO - Checking if environment pre-builds are      download.py:544
                             needed (this may take a while if building for the first time)
N E X T F L O W  ~  version 23.10.1
Launching `<PATH>/bactopia-3.0.1/main.nf` [spontaneous_bhaskara] DSL2 - revision: 0cd9f79ba7
WARN: Maximum memory (8) was adjusted to fit your system (7)
ERROR ~ <PATH>/bactopia-3.0.1/modules/nf-core/kraken2/params-cleanyerreads.json (No such file or directory)

 -- Check script '<PATH>/miniforge3/envs/bactopia/share/bactopia-3.0.1/./workflows/clean-yer-reads.nf' at line: 19 or see '.nextflow.log' file for more details
```

**Expected Behavior**
Completion of cleanyerreads workflow.

**Execution Environment**

- Bactopia Version: 3.0.1
- OS: Ubuntu 22.04.1 in WSL2
- Environment: bactopia in conda with Singularity

**Additional Information**
nextflow.log:
```
Apr-03 13:13:53.848 [main] DEBUG nextflow.cli.Launcher - $> nextflow run /home/pvh/miniforge3/envs/bactopia//share/bactopia-3.0.1/main.nf -w /home/pvh/Documents/code/SANBI/projects/cbio_teaching/vibrio/work/ --max_memory 8 --max_cpus 2 --max_retry 1 -profile singularity -resume -ansi-log false --wf cleanyerreads --samples sample-sheet.tsv --outdir cleaned_reads
Apr-03 13:13:53.939 [main] INFO  nextflow.cli.CmdRun - N E X T F L O W  ~  version 23.10.1
Apr-03 13:13:53.962 [main] DEBUG nextflow.plugin.PluginsFacade - Setting up plugin manager > mode=prod; embedded=false; plugins-dir=/home/pvh/.nextflow/plugins; core-plugins: nf-amazon@2.1.4,nf-azure@1.3.3,nf-cloudcache@0.3.0,nf-codecommit@0.1.5,nf-console@1.0.6,nf-ga4gh@1.1.0,nf-google@1.8.3,nf-tower@1.6.3,nf-wave@1.0.1
Apr-03 13:13:53.972 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Enabled plugins: []
Apr-03 13:13:53.973 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Disabled plugins: []
Apr-03 13:13:53.976 [main] INFO  org.pf4j.DefaultPluginManager - PF4J version 3.4.1 in 'deployment' mode
Apr-03 13:13:53.986 [main] INFO  org.pf4j.AbstractPluginManager - No plugins
Apr-03 13:13:54.004 [main] DEBUG nextflow.config.ConfigBuilder - Found config base: /home/pvh/miniforge3/envs/bactopia/share/bactopia-3.0.1/nextflow.config
Apr-03 13:13:54.006 [main] DEBUG nextflow.config.ConfigBuilder - Parsing config file: /home/pvh/miniforge3/envs/bactopia/share/bactopia-3.0.1/nextflow.config
Apr-03 13:13:54.031 [main] DEBUG nextflow.config.ConfigBuilder - Applying config profile: `singularity`
Apr-03 13:13:58.255 [main] DEBUG nextflow.config.ConfigBuilder - Available config profiles: [cfc_dev, uzl_omics, ifb_core, denbi_qbic, alice, mjolnir_globe, uppmax, incliva, ilifu, uge, rosalind_uge, gcp, lugh, unibe_ibu, vai, czbiohub_aws, jax, ccga_med, arcc_hawk, scw, unc_longleaf, tigem, tubingen_apg, google, apollo, ipop_up, vsc_calcua, pdc_kth, googlels, daisybio, eddie, medair, biowulf, apptainer, bi, bigpurple, adcra, cedars, pawsey_setonix, vsc_kul_uhasselt, pawsey_nimbus, ucl_myriad, utd_ganymede, icr_davros, ceres, munin, arm, rosalind, hasta, cfc, uzh, ebi_codon_slurm, ebc, ku_sund_dangpu, ccga_dx, crick, marvin, biohpc_gen, mana, york_viking, unc_lccc, wehi, awsbatch, wustl_htcf, arcc, imperial, maestro, software_license, utd_europa, standard, genotoul, nci_gadi, abims, janelia, nu_genomics, googlebatch, oist, sahmri, sge, mpcdf, leicester, vsc_ugent, create, sage, cambridge, jex, ebi_codon, cheaha, xanadu, nyu_hpc, test, computerome, ucd_sonic, seg_globe, sanger, dkfz, pasteur, ethz_euler, m3c, imb, ucl_cscluster, tuos_stanage, azurebatch, hki, crukmi, csiro_petrichor, qmul_apocrita, docker, engaging, gis, hypatia, psmn, eva, nygc, fgcz, crg, singularity, self_hosted_runner, tufts, uw_hyak_pedslabs, utd_sysbio, genouest, cbe, slurm, phoenix, seawulf, uod_hpc, fub_curta, uct_hpc, aws, aws_tower, binac]
Apr-03 13:13:58.306 [main] DEBUG nextflow.cli.CmdRun - Applied DSL=2 from script declararion
Apr-03 13:13:58.336 [main] INFO  nextflow.cli.CmdRun - Launching `/home/pvh/miniforge3/envs/bactopia/share/bactopia-3.0.1/main.nf` [spontaneous_bhaskara] DSL2 - revision: 0cd9f79ba7
Apr-03 13:13:58.337 [main] DEBUG nextflow.plugin.PluginsFacade - Plugins default=[]
Apr-03 13:13:58.337 [main] DEBUG nextflow.plugin.PluginsFacade - Plugins resolved requirement=[]
Apr-03 13:13:58.343 [main] DEBUG n.secret.LocalSecretsProvider - Secrets store: /home/pvh/.nextflow/secrets/store.json
Apr-03 13:13:58.346 [main] DEBUG nextflow.secret.SecretsLoader - Discovered secrets providers: [nextflow.secret.LocalSecretsProvider@11e17893] - activable => nextflow.secret.LocalSecretsProvider@11e17893
Apr-03 13:13:58.449 [main] DEBUG nextflow.Session - Session UUID: 5c19cb3c-9023-4b80-8a91-6168e9cbe857
Apr-03 13:13:58.449 [main] DEBUG nextflow.Session - Run name: spontaneous_bhaskara
Apr-03 13:13:58.450 [main] DEBUG nextflow.Session - Executor pool size: 8
Apr-03 13:13:58.461 [main] DEBUG nextflow.file.FilePorter - File porter settings maxRetries=3; maxTransfers=50; pollTimeout=null
Apr-03 13:13:58.513 [main] DEBUG nextflow.cli.CmdRun -
  Version: 23.10.1 build 5891
  Created: 12-01-2024 22:01 UTC (13-01-2024 00:01 SAST)
  System: Linux 5.15.146.1-microsoft-standard-WSL2
  Runtime: Groovy 3.0.19 on OpenJDK 64-Bit Server VM 21-internal-adhoc.conda.src
  Encoding: UTF-8 (UTF-8)
  Process: 219060@ntiffin-22 [127.0.1.1]
  CPUs: 8 - Mem: 7.6 GB (4.8 GB) - Swap: 2 GB (1.4 GB) - Virtual threads ON
Apr-03 13:13:58.557 [main] DEBUG nextflow.Session - Work-dir: /home/pvh/Documents/code/SANBI/projects/cbio_teaching/vibrio/work [ext2/ext3]
Apr-03 13:13:58.576 [main] DEBUG nextflow.executor.ExecutorFactory - Extension executors providers=[]
Apr-03 13:13:58.593 [main] DEBUG nextflow.Session - Observer factory: DefaultObserverFactory
Apr-03 13:13:58.665 [main] DEBUG nextflow.cache.CacheFactory - Using Nextflow cache factory: nextflow.cache.DefaultCacheFactory
Apr-03 13:13:58.684 [main] DEBUG nextflow.util.CustomPoolFactory - Creating virtual thread pool
Apr-03 13:13:59.059 [main] DEBUG nextflow.Session - Session start
Apr-03 13:13:59.065 [main] DEBUG nextflow.trace.TraceFileObserver - Workflow started -- trace file: /home/pvh/Documents/code/SANBI/projects/cbio_teaching/vibrio/cleaned_reads/bactopia-runs/cleanyerreads-20240403-131357/nf-reports/cleanyerreads-trace.txt
Apr-03 13:13:59.071 [main] DEBUG nextflow.Session - Using default localLib path: /home/pvh/miniforge3/envs/bactopia/share/bactopia-3.0.1/lib
Apr-03 13:13:59.074 [main] DEBUG nextflow.Session - Adding to the classpath library: /home/pvh/miniforge3/envs/bactopia/share/bactopia-3.0.1/lib
Apr-03 13:13:59.074 [main] DEBUG nextflow.Session - Adding to the classpath library: /home/pvh/miniforge3/envs/bactopia/share/bactopia-3.0.1/lib/nfcore_external_java_deps.jar
Apr-03 13:13:59.200 [main] DEBUG nextflow.script.ScriptRunner - > Launching execution
Apr-03 13:14:01.754 [main] WARN  nextflow.Nextflow - Maximum memory (8) was adjusted to fit your system (7)
Apr-03 13:14:01.797 [main] DEBUG nextflow.script.ScriptRunner - Parsed script files:
  Script_f447eb07a737d27b: /home/pvh/miniforge3/envs/bactopia/share/bactopia-3.0.1/./workflows/../lib/nf/functions.nf
  Script_aa86a8c70df9b76f: /home/pvh/miniforge3/envs/bactopia/share/bactopia-3.0.1/./workflows/clean-yer-reads.nf
  Script_28b956a88b5bebd6: /home/pvh/miniforge3/envs/bactopia/share/bactopia-3.0.1/main.nf
  Script_5006a0b829d2df9f: /home/pvh/miniforge3/envs/bactopia/share/bactopia-3.0.1/./workflows/../lib/nf/bactopia.nf
Apr-03 13:14:01.797 [main] DEBUG nextflow.Session - Session aborted -- Cause: /home/pvh/miniforge3/envs/bactopia/share/bactopia-3.0.1/modules/nf-core/kraken2/params-cleanyerreads.json (No such file or directory)
Apr-03 13:14:02.053 [main] ERROR nextflow.cli.Launcher - /home/pvh/miniforge3/envs/bactopia/share/bactopia-3.0.1/modules/nf-core/kraken2/params-cleanyerreads.json (No such file or directory)
java.io.FileNotFoundException: /home/pvh/miniforge3/envs/bactopia/share/bactopia-3.0.1/modules/nf-core/kraken2/params-cleanyerreads.json (No such file or directory)
        at java.base/java.io.FileInputStream.open0(Native Method)
        at java.base/java.io.FileInputStream.open(FileInputStream.java:213)
        at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152)
        at groovy.util.CharsetToolkit.<init>(CharsetToolkit.java:78)
        at org.codehaus.groovy.runtime.ResourceGroovyMethods.newReader(ResourceGroovyMethods.java:1780)
        at org.codehaus.groovy.runtime.ResourceGroovyMethods.getText(ResourceGroovyMethods.java:591)
        at org.codehaus.groovy.runtime.dgm$1068.doMethodInvoke(Unknown Source)
        at org.codehaus.groovy.reflection.GeneratedMetaMethod$Proxy.doMethodInvoke(GeneratedMetaMethod.java:83)
        at groovy.lang.MetaClassImpl.getProperty(MetaClassImpl.java:1942)
        at groovy.lang.MetaClassImpl.getProperty(MetaClassImpl.java:3815)
        at groovy.lang.DelegatingMetaClass.getProperty(DelegatingMetaClass.java:126)
        at org.codehaus.groovy.runtime.InvokerHelper.getProperty(InvokerHelper.java:195)
        at org.codehaus.groovy.runtime.callsite.PojoMetaClassGetPropertySite.callGetProperty(PojoMetaClassGetPropertySite.java:44)
        at NfcoreSchema.validateParameters(NfcoreSchema.groovy:35)
        at NfcoreSchema$validateParameters.call(Unknown Source)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:166)
        at WorkflowMain.initialise(WorkflowMain.groovy:122)
        at WorkflowMain$initialise.call(Unknown Source)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:166)
        at Script_aa86a8c70df9b76f.runScript(Script_aa86a8c70df9b76f:19)
        at nextflow.script.BaseScript.run0(BaseScript.groovy:144)
        at nextflow.script.BaseScript.run(BaseScript.groovy:192)
        at nextflow.script.ScriptParser.runScript(ScriptParser.groovy:236)
        at nextflow.script.ScriptParser.runScript(ScriptParser.groovy:222)
        at nextflow.script.IncludeDef.memoizedMethodPriv$loadModule0PathMapSession(IncludeDef.groovy:151)
        at nextflow.script.IncludeDef.access$0(IncludeDef.groovy)
        at nextflow.script.IncludeDef$__clinit__closure2.doCall(IncludeDef.groovy)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
        at java.base/java.lang.reflect.Method.invoke(Method.java:580)
        at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
        at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
        at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1030)
        at groovy.lang.Closure.call(Closure.java:427)
        at org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)
        at org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)
        at org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)
        at org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)
        at nextflow.script.IncludeDef.loadModule0(IncludeDef.groovy)
        at nextflow.script.IncludeDef.load0(IncludeDef.groovy:123)
        at nextflow.script.IncludeDef$load0$1.call(Unknown Source)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:139)
        at Script_28b956a88b5bebd6.runScript(Script_28b956a88b5bebd6:10)
        at nextflow.script.BaseScript.run0(BaseScript.groovy:144)
        at nextflow.script.BaseScript.run(BaseScript.groovy:192)
        at nextflow.script.ScriptParser.runScript(ScriptParser.groovy:236)
        at nextflow.script.ScriptRunner.run(ScriptRunner.groovy:242)
        at nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:137)
        at nextflow.cli.CmdRun.run(CmdRun.groovy:372)
        at nextflow.cli.Launcher.run(Launcher.groovy:500)
        at nextflow.cli.Launcher.main(Launcher.groovy:672)
```

**Comment by rpetit3** [comment]
Hi @pvanheus,

Thank you for reporting, this should be fixed in my [dev version](https://github.com/bactopia/bactopia/commit/018f1899f97f72c2256a9ac6337ebcbd244369fb#diff-0c3ef8bce36ac571025c398963862bdafba71a968cbf7c36f5a12224faabb0e8).

I can push a patch release soon if you think its needed sooner than later. Just let me know.

Cheers,
Robert

**Comment by rpetit3** [comment]
Hi @pvanheus 

I've released v3.1.0 (https://github.com/bactopia/bactopia/releases/tag/v3.1.0)

Will go ahead and close,
Robert
